ocrd/ocrd/workspace.py

```diff
@@ -836,6 +836,7 @@ def image_from_segment(self, segment, parent_image, parent_coords,
     def save_image_file(self, image,
                         file_id,
                         file_grp,
+                        dpi=None,
```

Comment:
The position of the new kwarg makes `test_save_image_file` (which does not use keys) fail.

Reply:
552a7c9 test still fails because file isn't found but that's another issue, see below.